src/compass/defaults/s1_cslc_radar.yaml

```
@@ -48,6 +48,9 @@ runconfig:
            resample:
                lines_per_block: 1000
                flatten: False
+           correction_luts:
+               x_spacing: 500
+               y_spacing: 50
```

```suggestion
               # LUT spacing in x/slant range direction in meters
               x_spacing: 500
               # LUT spacing in y/azimuth direction in seconds
               y_spacing: 50
```
Adding documentation.